index.html

```diff
@@ -1611,7 +1601,8 @@ <h3>
             <dfn data-dfn-for="icon purpose">any</dfn> (default)
           </dt>
           <dd>
-            The user agent is free to display the icon in any context.
+            The user agent is free to display the icon where a more specialized
```

Comment:
Discussion idea: `The user agent can present this icon when a specific purpose is not required.`